contrib/google.golang.org/grpc/grpc_test.go

```diff
@@ -1203,3 +1209,82 @@ func BenchmarkUnaryServerInterceptor(b *testing.B) {
 		}
 	})
 }
+
+type roundTripper struct {
+	assertSpanFromRequest func(r *http.Request)
+}
+
+func (rt *roundTripper) RoundTrip(r *http.Request) (*http.Response, error) {
+	rt.assertSpanFromRequest(r)
+	return http.DefaultTransport.RoundTrip(r)
+}
+
+func TestRegressionServiceNames(t *testing.T) {
```

Comment:
`Regression` here means that this test is a regression test of a known bug? If so, consider filing an issue and naming it `TestIssueXXXX` instead, to help keep the context

Reply:
Good idea, issue filed #2050 and name updated